raiden/ui/app.py

```diff
@@ -247,6 +248,27 @@ def run_app(
     known_given_network_id = given_network_id in ID_TO_NETWORKNAME
     known_node_network_id = node_network_id in ID_TO_NETWORKNAME
 
+    if config['services']['pathfinding_service_address'] is None:
+        click.secho(
```

Comment:
I'm not sure if we want to display this right now, as we still aim to be backwards compatible. Maybe only do this in the development environment?

Reply:
Good point. When I understood Rakan correctly this should be displayed, but Raiden should keep on going anyway. Isn't it backwards compatible when we keep this? Alternatively, I could simply comment it out for now?

Reply:
I am ok with keeping this as it is not related to backwards-compatibility.